datasquare.go

```diff
@@ -198,12 +198,20 @@ func (ds *dataSquare) computeRoots() {
 
 		go func(i uint) {
 			defer wg.Done()
-			rowRoots[i] = ds.getRowRoot(i)
+			rowRoot, err := ds.getRowRoot(i)
+			if err != nil {
+				panic(err)
+			}
+			rowRoots[i] = rowRoot
 		}(i)
 
 		go func(i uint) {
 			defer wg.Done()
-			colRoots[i] = ds.getColRoot(i)
+			colRoot, err := ds.getColRoot(i)
+			if err != nil {
+				panic(err)
```

Comment:
I know this probably shouldn't get hit and its in a goroutine, but is there a way to bubble these up as well? computing the roots is

Reply:
errgroup

Reply:
Thanks for the tip @Wondertan ! Resolved in https://github.com/celestiaorg/rsmt2d/pull/157/commits/bbacca88a5448fa16df25f9aeeece7e86c535fff